exporters/metric/cortex/README.md

```diff
@@ -21,15 +21,15 @@ if err != nil {
 
 ## Configuration
 
-The Exporter needs certain information, such as the endpoint URL and push interval
-duration, to function properly. This information is stored in a `Config` struct, which is
-passed into the Exporter during the setup pipeline. 
+The Exporter needs certain information, such as the endpoint URL and push interval duration, to
+function properly. This information is stored in a `Config` struct, which is passed into the
+Exporter during the setup pipeline.
 
 ### Creating the Config struct
 
-Users can either create the struct manually or use a `utils` submodule in the package to
-read settings from a YAML file into a new Config struct using `Viper`. Here are the
-supported YAML properties as well as the Config struct that they map to.
+Users can either create the struct manually or use a `utils` submodule in the package to read
+settings from a YAML file into a new Config struct using `Viper`. Here are the supported YAML
```

Comment:
Will users know what "Viper" is? Or is worth turning the name into a link to relevant docs/code on that?

Reply:
Viper is pretty common, but I think adding a link would be helpful.